docs/source/peers/peers.md

```diff
@@ -323,14 +323,15 @@ a request to the Fabric Gateway service to evaluate a transaction proposal.
 
 The target peer, selected by the client application, executes the transaction
 by invoking chaincode --- this step can be described as simulating the transaction,
-because it runs the transaction without any effect on the ledger. The peer then
-returns its transaction result to the client.  
+because it runs the transaction without any effect on the ledger. The target peer then
+returns its proposal response to the gateway service.
```

Comment:
this is if you are using the gateway, which is a client in a sense, but you don't have to, hence:

```suggestion
returns its proposal response to the client.
```

Reply:
This whole page has been rewritten in v2.4 to reflect that all transaction invocations would be done via the gateway service (the old SDKs will be deprecated soon).  So, from that point of view, the proposal response is returned to the gateway service.  However, this whole paragraph doesn't read very well in the context of what precedes it and what follows it.  I'd suggest rewording this paragraph as follows:
```
The target (gateway) peer, selected by the client application, executes the transaction
by invoking chaincode (this can be described as simulating the transaction,
because it runs the transaction without any effect on the ledger).   
The gateway service uses this initial invocation to analyse the actions of the 
chaincode and derive an endorsement policy for the transaction.
```
The next paragraph then goes on to say that it uses this endorsement policy to contact other peers and collect their responses.